common/service/service.go

```diff
@@ -124,6 +127,9 @@ func New(params *BootstrapParams) Service {
 		dispatcherProvider:    params.DispatcherProvider,
 		dynamicCollection:     dynamicconfig.NewCollection(params.DynamicConfig, params.Logger),
 	}
+
+	tlRPS := sVice.dynamicCollection.GetIntProperty(dynamicconfig.ThrottledLogRPS, defaultThrottledLogRPS)
```

Comment:
It would be more consistent if we can put that dynamic config in each service config, initialize in "NewConfig" and put the default value there. 
A good example is that we have history/matching/frontendRPS. 
That will also give us flexibility to give different service different quotas.